docs/release-notes/RELEASE-NOTES-1.4.0.md

```diff
@@ -0,0 +1,107 @@
+# About this release
+
+[This][this-release] is a maintenance release.
+It:
+* Does all the things mentioned temporarily in files [/docs/release-notes/next/PT-*.md](/docs/release-notes/next/).
+
+TODO: When preparing the release, concatenate all `/docs/release-notes/next/*` Markdown files and place them in this file. (Hint: you can use auxiliary script `scripts/cat-files-in-directory-sorted-by-committer-date` and command `git log -p -w --color-moved`.)
+
+[this-release]: https://github.com/aeternity/aeternity/releases/tag/v1.4.0
+
+This release is backward compatible with `v1.3.*`, `v1.2.*`, `v1.1.*` and `v1.0.*`.
```

Comment:
In future can be changed to `This release is backward compatible with all `v1.*` releases.